legate/core/launcher.py

```diff
@@ -600,6 +605,12 @@ def add_output(self, store, proj, tag=0, flags=0):
             self._outputs, store, proj, Permission.WRITE, tag, flags
         )
 
+    # currently this is adding to outputs but we can have a seperate "temps" array in the core
+    def add_temp(self, store, proj, tag=0, flags=0):
+        self.add_store(
+            self._outputs, store, proj, Permission.WRITE, tag, flags
+        )
+
```

Comment:
todo: remove
I was thinking of adding a notion of a 'temporary' that could be re-used. Didn't end up getting to this